src/interfaces/peers.py

```diff
@@ -29,7 +33,12 @@ def validate_peer_exists(vpn_name: str, ip_address: str, vpn_manager) -> None:
 
 
 @peer_router.get("/vpn/{vpn_name}/peers", tags=["peers"], response_model=list[PeerResponseModel])
-def get_peers(vpn_name: str, hide_secrets: bool = True) -> list[PeerResponseModel]:
+def get_peers(
+    vpn_name: str = Path(
+        ..., regex="^[A-Za-z0-9_-]+$", description="Only alphanumeric characters and - _ are allowed in the VPN name."
+    ),
+    hide_secrets: bool = True,
+) -> list[PeerResponseModel]:
```

Comment:
You only need to make this distinction on the endpoint where you add or update a vpn.  The VPN won't be found if they try and use any of the invalid characters, which is fine.  The main reason to take it off, however, is because it makes the OpenAPI spec harder to read.
```suggestion
def get_peers(
    vpn_name: str = Path(..., description="The name of the VPN the peer is connected to."),
    hide_secrets: bool = True,
) -> list[PeerResponseModel]:
```